src/database/models/usersBrdgames.ts

```diff
@@ -64,3 +64,5 @@ User.belongsToMany(BrdGame, {
   through: UsersBrdgames,
   foreignKey: "userId"
 });
+
+const foo = new UsersBrdgames();
```

Comment:
@Alexandria Did you mean to leave this part in?

Reply:
Nope, sure didn't  😅